src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,117 @@
+.cards-container {
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
```

Comment:
the card looks like a separate block to me. Why don't we make it a separate file as well?
This approach is usually used for making styling easier, it's gonna be easier to navigate between the pieces of your code if you need to change something